windows/installer.nsi

```diff
@@ -370,6 +402,11 @@ Function FinishPage.Show ; set the user choices if they were remembered
 
     ShowWindow $mui.FinishPage.Run 1
 
+    !if ${BUILD_OPTION} != "jackonwindows"
+        ;Add ASIO label to dialog when not using JACK
```

Comment:
```suggestion
        ; Add ASIO label to dialog when not using JACK
```

Reply:
Rebase done, white space added and force-pushed. Thanks for spotting that.